.eslintrc.yaml

```diff
@@ -110,6 +110,9 @@ rules:
   }, {
     selector: "ThrowStatement > CallExpression[callee.name=/Error$/]",
     message: "Use new keyword when throwing an Error."
+    }, {
+    selector: "CallExpression[callee.object.name='assert'][callee.property.name='fail'][arguments.length=1]",
```

Comment:
Thanks for the detail, definitely not really related to this PR anyway.

Yeah, I find the idea that you have to have the third argument as `falsy` to actually use the fourth argument a bit hard to wrap my head around.